PeakCoreData/Helpers/NSManagedObjectContext.swift

```diff
@@ -44,4 +44,29 @@ extension NSManagedObjectContext {
         
         return try _helper(fn: performAndWait(_:), execute: block, rescue: { throw $0 } )
     }
+    
+    /**
+     Batch deletes all objects for all entities.
+     
+     - Note: This method cannot be unit tested because it is incompatible with `NSInMemoryStoreType`.
+     
+     - parameter context:   The context to use.
+     */
+    static func batchDeleteAllEntities() {
```

Comment:
You've made this a static function on `NSManagedObjectContext`, which means it can't access instance member `persistentStoreCoordinator`. Have you run this on some data to test it works?

Reply:
This was a copy-paste error. This function should not be static.